### DISCUSSION TITLE
Approach for using groups in functions in tight loops?

### DISCUSSION THREAD

**Discussion by Net5F** [skypjack/entt#810] [Q&A] [answered]
Hello, I often end up having somewhat-tight loops that access components through groups. When I go to refactor code from these loops out into functions, I hit the issue of not knowing how to pass the group around.

Is the answer just to re-construct the group in every function? Or is there some type that I can use for the parameter?

**Comment by skypjack** [comment, marked as answer]
You can safely reconstruct groups in every function actually. The cost is that of looking up the pools in the registry and that's it, therefore pretty much negligible.

**Reply by Net5F** [reply]
Good to hear, thanks!
